pkg/apis/serving/v1alpha1/revision_validation.go

```diff
@@ -118,24 +131,73 @@ func ValidateContainerConcurrency(cc RevisionContainerConcurrencyType, cm Revisi
 	return nil
 }
 
-func validateContainer(container corev1.Container) *apis.FieldError {
+func validateVolume(volume corev1.Volume) *apis.FieldError {
+	var errs *apis.FieldError
+	if volume.Name == "" {
```

Comment:
Should we also have validation for valid name (length and character set) here so that it is caught before it gets to our MakePodSpec in the controller?